payment_tenpay/README.md

```diff
@@ -0,0 +1,8 @@
+# payment_tenpay
```

Comment:
no README.md
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst